### PR TITLE
QBO - Delete Journal entries

### PR DESCRIPTION
**Pull request by d-laing**
# Description

This pull request adds a new article on deleting Journal entries from QuickBooks Online.

## Type of change

Please delete options that are not relevant.

- [x] New document(s)/updating existing

## Reviews and merging

You are responsible for getting your PR merged. Address review comments promptly and **make sure to merge the PR when ready**.
Feel free to 'Enable automerge' - your PR will automatically merge when accepted and passing the build.

### CODE REVIEW

File: docs/integrations/accounting/netsuite/accounting-netsuite-delete-bills-billpayments.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ The _Delete bill_ and _Delete bill payment_ endpoints allow you to delete specif
 
    :::
 
-2. [List the push operations](/codat-api#/operations/get-companies-companyId-push) for the company. A `Success` status for the push operation means the Bill or Bill payment object was deleted from NetSuite.
```
Comment (d-laing) [inline docs/integrations/accounting/netsuite/accounting-netsuite-delete-bills-billpayments.md:29]: Also searched for any more uses of this endpoint.

File: docs/integrations/accounting/quickbooksonline/accounting-quickbooksonline-delete-journal-entries.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+---
+title: "Delete Journal entries from QuickBooks Online"
+description: "Learn how to delete Journal entries from QuickBooks Online using the deletion endpoint"
+---
+
+The (TBC) [Delete Journal entries](/accounting-api#/) endpoint allows you to delete a specified Journal entry from QuickBooks Online (QBO).
```
Comment (d-laing) [inline docs/integrations/accounting/quickbooksonline/accounting-quickbooksonline-delete-journal-entries.md:6]: To add.

Comment (mcclowes) [reply docs/integrations/accounting/quickbooksonline/accounting-quickbooksonline-delete-journal-entries.md:6]: Delete for now then??

Comment (pzaichkina) [reply docs/integrations/accounting/quickbooksonline/accounting-quickbooksonline-delete-journal-entries.md:6]: yeah I agree with Max - shouldn't really have a tbc in live docs?
also, do we need to have the (QBO) abbreviation when it's like the fourth page on QBO in a list?

File: docs/integrations/accounting/quickbooksonline/accounting-quickbooksonline-delete-journal-entries.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+:::danger Use with caution
+
+The underlying QuickBooks API endpoint allows users to delete any object from their QBO instance. Make sure you specify the correct ID of the Journal entry you want to delete.
```
Comment (pzaichkina) [inline docs/integrations/accounting/quickbooksonline/accounting-quickbooksonline-delete-journal-entries.md:10]: do we want to say something like "This action is not reversible"? (if it's actually the case)

Comment (d-laing) [reply docs/integrations/accounting/quickbooksonline/accounting-quickbooksonline-delete-journal-entries.md:10]: Yep, I think so. I will double check if that's the case.

File: docs/integrations/accounting/quickbooksonline/accounting-quickbooksonline-delete-journal-entries.md (hunk 2 above)
Comment (d-laing) [reply docs/integrations/accounting/quickbooksonline/accounting-quickbooksonline-delete-journal-entries.md:6]: Good catch, I'll remove the TBC for now.

Yeah, I see your point about QBO. It's probably enough to spell out the abbreviation on the landing page only, not in the sub-pages.

File: docs/integrations/accounting/quickbooksonline/accounting-quickbooksonline-delete-journal-entries.md (hunk 3 above)
Comment (d-laing) [reply docs/integrations/accounting/quickbooksonline/accounting-quickbooksonline-delete-journal-entries.md]: I added the sentence, "deleting objects from QBO is not reversible".

**Review by pzaichkina — APPROVED** [review]

**Pull request merged**